array-filter/main.js

```diff
@@ -0,0 +1,26 @@
+const numbers = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+
+const names = [
+  'Ada',
+  'Hedy',
+  'Jean',
+  'Grace',
+  'Evelyn',
+  'Joan',
+  'Elizabeth',
+  'Janese',
+  'Donna'
+];
+
+// Set up an object to keep track of filtered arrays
+const allArrays = {
+  evenNumbers: numbers.filter((num) => num % 2 === 0),
+  overFive: numbers.filter((num) => num > 5),
+  startWithE: names.filter((name) => name[0].toLowerCase() === 'e'),
```

Comment:
The spec actually says that it starts with 'E' and you catch both 'e' and 'E'.

Reply:
I was assuming the user input may have inconsistent formatting, so I wanted to account for both. Perhaps I should have made it more strict to follow the prompt!